digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/dao/RoadwayChangesDAOSpec.scala

```diff
@@ -387,4 +387,54 @@ class RoadwayChangesDAOSpec extends FunSuite with Matchers {
       roadNames.head should equal("Turku-Oulu")
     }
   }
+
+  test("When fetching type termination road address change info for road address browser then return the termination change info") {
```

Comment:
Urrr..  "type termination road address change info" :D
Menee kauheesti aikaa tän ymmärtämiseen

Yritän jotain...
Katsokaa, ymmärrätekö te puolestanne tästä mitään:
test("When fetching road address change info for a terminated road (-> type of change: termination)  for the road address browser
      then return the change info of the terminated layer")